include/ofi_hmem.h

```diff
@@ -158,6 +158,19 @@ int cuda_get_handle(void *dev_buf, void **handle);
 int cuda_open_handle(void **handle, uint64_t device, void **ipc_ptr);
 int cuda_close_handle(void *ipc_ptr);
 int cuda_get_base_addr(const void *ptr, void **base, size_t *size);
+
+/*
+ * This enum lists all the poissble situations about how
+ * the environment variable FI_HMEM_CUDA_ENABLE_XFER was set.
+ */
+enum cuda_xfer_setting
+{
+	CUDA_XFER_DISABLED = 0, /* FI_HMEM_CUDA_ENABLE_XFER set to 0/false/no */
+	CUDA_XFER_ENABLED = 1, /* FI_HMEM_CUDA_ENABLE_XFER set to 1/true/yes */
+	CUDA_XFER_UNSPECIFIED = 2, /* FI_HMEM_CUDA_ENABLE_XFER was not set */
+};
```

Comment:
It's better for an enum if unspecified = 0, which will be the default value for any static variable, and aligns with other libfabric values where 0 indicates not set.

Reply:
fixed in new revision